docs/contracts/inheritance.rst

```diff
@@ -172,6 +176,48 @@ virtual method lookup.
 
 .. _constructor:
 
+Function Overriding
```

Comment:
I think it would be also good to add some documentation on modifier inheritance here https://solidity.readthedocs.io/en/v0.5.12/structure-of-a-contract.html#function-modifiers.